### ISSUE TITLE
Converting sparse Index slices to a dense Tensor of unknown shape..

### ISSUE THREAD

**Issue by presterman** [keras-team/keras#11231]
I am running the code noted below on my own machine (Macbook Pro v 10.13.2. memory 8 GB 2133 MHz LPDDR3) 
It is the imdb example from this page:
https://www.tensorflow.org/tutorials/keras/basic_text_classification

I've changed the code a little, viz:
model.compile(optimizer=keras.optimizers.Adam(lr=0.001, beta_1=0.9, beta_2=0.999, epsilon=None, decay=0.0, amsgrad=False),loss='binary_crossentropy',metrics=['accuracy'])

instead of:
model.compile(optimizer=tf.train.AdamOptimizer(),
              loss='binary_crossentropy',
              metrics=['accuracy'])

Then, I've saved the model as a h5 file using:
 model.save('model_name')

In a new file I load the model using:
 model = keras.models.load_model('model_name'') and then call model.predict('something')
The program stops running and I always get this error:

'/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/tensorflow/python/ops/gradients_impl.py:108: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.
  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "'

How should I proceed? I know from looking at stackoverflow etc. that this issue can be addressed in tensorflow itself, but is there a keras solution? or do I need to start digging into tensorflow and start changing things there myself?

Thanks.
